messaging.rst

```diff
@@ -17,34 +17,48 @@ Data required by the smart contracts to update the payment channel end of the pa
-+=======================+=============+===============================================================================+
-|  nonce                | uint64      | Strictly monotonic nonce                                                      |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-|  transferred_amount   | uint256     | counter of tokens sent                                                        |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-|  locksroot            | bytes32     | Root of merkle tree of all pending lock lockhashes                            |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-|  channel_identifier   | uint256     | Channel identifier inside the TokenNetwork contract                           |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-| token_network_address | address     | Address of the TokenNetwork contract                                          |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-| chain_id              | uint256     | Chain identifier as defined in EIP155                                         |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-|  additional_hash      | bytes32     | Hash of additional data used on the application layer, e.g.: payment metadata |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
-|  signature            | bytes       | An elliptic curve 256k1 signature                                             |
-+-----------------------+-------------+-------------------------------------------------------------------------------+
++------------------------+------------+--------------------------------------------------------------+
+| Field Name             | Field Type |  Description                                                 |
++========================+============+==============================================================+
+|  balance_hash          | bytes32    | Balance data hash                                            |
++------------------------+------------+--------------------------------------------------------------+
+|  nonce                 | uint64     | Strictly monotonic value used to order transfers             |
++------------------------+------------+--------------------------------------------------------------+
+|  additional_hash       | bytes32    | Computed from the message. Used for message authentication   |
```

Comment:
Definition should be kept as `Hash of additional data used on the application layer, e.g.: payment metadata`